utilities.c

```diff
@@ -301,7 +301,7 @@ char * iio_getenv (char * envvar)
 	len = strnlen(hostname, tmp);
 
 	/* Should be smaller than max length */
-	if (len <= tmp)
+	if (len > tmp)
```

Comment:
should this be `len >= tmp`?
`len > tmp` should be an impossible condition

Reply:
Oh yeah, you are right. It could even be `len == tmp` since it should never be `>` than `tmp`

Reply:
yep; `len == tmp` also works